main.rb

```diff
@@ -27,21 +39,19 @@ def initialize
   end
 
   def run
-    @retrieve_data.retrieve_all(@game_manager, @music_album_manager, @book_manager)
-    puts ''
-    puts 'WELCOME TO MY-CATALOG APP'
-    puts ''
+    @retrieve_data.retrieve_all(@game_manager, @music_album_manager)
```

Comment:
On Startup there is an Argument error, seems like you deleted the @book_manager argument